### ISSUE TITLE
Resolve AWS non compliances

### ISSUE THREAD

**Issue by ofanobilbao** [ebi-ait/hca-ebi-dev-team#374]
Yesterday, 14/01/2021, I checked the AWS console while doing some post-invoicing checks. I found that there are once again quite a lot of non compliances in the system. This needs to be resolved at least to the same level we did back in October.

**Comment by ofanobilbao** [comment]
Yesterday, 14/01/2021, I checked the AWS console while doing some post-invoicing checks. I found that there are once again quite a lot of non compliances in the system. This needs to be resolved at least to the same level we did back in October.

**Comment by lauraclarke** [comment]
@prabh-t @aaclan-ebi Can you look into this? Does this need an SOP and/or a regular task in the list?

**Comment by clairerye** [comment]
I understand that we have been asked to move to a dedicated HCA AWS account which will then make billing straight forward. Because of that can this be ignored?

**Comment by prabh-t** [comment]
@ofanobilbao can you let us know where you're looking at for these non compliances? I can't see any hca-specific ones if I am looking at the right place. In any case, it is unlikely we look into tags since we will be moving to a new hca aws account.

**Comment by lauraclarke** [comment]
Presumably, this needs to be resolved to enable accurate billing for January (and February depending on when we make the switch to the new account)

**Comment by ofanobilbao** [comment]
I look at this same way as before, from the AWS console and the config setting @prabh-t . I would confirm with the cloud consultants if you like. But as Laura points out we would like the billing up to the move to be accurate. I think it's the right approach. But like I said feel free to check with the cloud consultants.

**Comment by prabh-t** [comment]
I see a lot of new rules which are not filtered for hca resources. There used to be one rule for hca which I can't see anymore. The non-compliances shown in the dashboard are from all resources in the ebi aws account.

**Comment by lauraclarke** [comment]
Okay, it sounds like this needs to be followed up with the cloud team to understand if these issues are created by us or if other services have been added to this account

**Comment by prabh-t** [comment]
On a related note, the cost saving measures implemented some time back seem to be showing some effect: 
https://console.aws.amazon.com/billing/home#/budgets/details?name=HCA%20Budgets

**Comment by ofanobilbao** [comment]
Yes, I have seen that the measures are having an impact, although December being also a quieter month I did not want to comment on that just yet. Please, check with the cloud consultants directly so that I don't act as a bridge. My understanding was that most of what we could see was our non compliances. And I wanted to make sure that we were not neglecting them. I hope that makes sense.
